pages/api/slack-app-manifest.ts

```diff
@@ -7,11 +7,13 @@ import { Response } from "retes/response";
 const handler: Handler = () => {
   const manifest = {
     display_information: {
-      name: "Saleor bot",
+      name: "Saleor",
+      description: "Receive new orders to your Slack channel",
```

Comment:
```suggestion
      description: "Receive notifications to your Slack channel",
```

We'll introduce different notifications in the future, so we can be more generic here.